cmd/api-router.go

```diff
@@ -34,7 +34,7 @@ func registerAPIRouter(mux *router.Router) {
 	}
 
 	// API Router
-	apiRouter := router.NewRouter().PathPrefix("/").Subrouter()
+	apiRouter := mux.NewRoute().PathPrefix("/").Subrouter()
```

Comment:
Also ping me on how to reproduce it will see a cleaner fix. The existing route is wrong and can lead to bugs.